lab_1/main.py

```diff
@@ -125,4 +207,44 @@ def sort_concordance(tokens: list, word: str, left_context_size: int, right_cont
     left_sort = True
     --> [['dog', 'is', 'happy', 'but', 'the', 'cat'], ['man', 'is', 'happy', 'the', 'dog', 'is']]
     """
-    pass
+    conco_all = []
+    index_word = 0
+    left_context_bool, right_context_bool = False, False
+    if isinstance(left_sort, bool):
```

Comment:
Usually, there are checks for input arguments and a function body - separated.
Try to split them and get rid of nesting.